setup.py

```diff
@@ -16,18 +16,15 @@
 
 from setuptools import setup, find_packages
 
-dependencies = [
-    "py4j==0.10.8.1",
-    "pytest==5.1.3",
-    "mlflow==1.6.0",
-    "mleap==0.15.0",
-    "graphviz==0.13",
-    "future"
-]
+REQUIREMENTS_FILE = "requirements.txt"
+
+with open(REQUIREMENTS_FILE, "r") as dependencies_file:
+    DEPENDENCIES = dependencies_file.readlines()
+
 setup(
     name="splicemachine",
-    version="1.0.10",
-    install_requires=dependencies,
+    version="2.0.0",
```

Comment:
I see that this is intended as a breaking change? That would be good to discuss with the team and will have implications during the next production upgrade.

Reply:
@njnygaard Thanks, and yes we are doing an overview tomorrow as part of our hackathon